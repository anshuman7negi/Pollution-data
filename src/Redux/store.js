import { configureStore } from '@reduxjs/toolkit';
import PollutionReducer from './PollutionSlice';

const store = configureStore({
  reducer: {
    pollution: PollutionReducer,
  },
});

export default store;
