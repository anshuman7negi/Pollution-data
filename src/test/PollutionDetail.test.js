import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../Redux/store';
import Pollution from '../components/PollutionDetail';

test('Pollution component matches snapshot', () => {
  const { asFragment } = render(
    <Provider store={store}>
      <Router>
        {' '}
        {}
        <Pollution />
      </Router>
    </Provider>,
  );

  expect(asFragment()).toMatchSnapshot();
});
