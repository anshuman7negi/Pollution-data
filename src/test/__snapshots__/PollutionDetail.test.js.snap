// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pollution component matches snapshot 1`] = `
<DocumentFragment>
   
  <p>
    Loading...
  </p>
</DocumentFragment>
`;
